data/human/human missions.txt

```diff
@@ -3719,11 +3719,64 @@ mission "Earth Retirement"
 			`	As you count up the large clump of credits you were handed (the sum comes out to the expected <payment>), they slowly make their way to the train station. Presumably, their new apartment is far from here.`
 				accept
 			label reject
+			action
+				event "charles donna timer" 90
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
 
 
 
+event "charles donna timer"
+
+
+
+mission "Earth Retirement 2"
+	name "Retired Family to Shangri-La"
+	description "Transport two seniors and their son to <destination>, where they plan to retire. They will pay you <payment>."
+	minor
+	source "Earth"
+	destination "Shangri-La"
+	to offer
+		has "event: charles donna timer"
+		has "outfit: Luxury Accommodations"
+	passengers 3
+	on offer
+		conversation
+			`As you enter the spaceport, you see a familiar sight. The elderly couple, Charles and Donna, who you helped move to Earth some months ago are talking to a young well-dressed man. Donna looks much the same as you remember her, but Charles is sporting a bandage on his head, and one of his arms looks like it's in a sling. When they see you, Donna points excitedly in your direction, and the entire group moves to meet you.`
```

Comment:
```suggestion
			`As you enter the spaceport, you see a familiar sight. The elderly couple, Charles and Donna Singer, you helped move to Earth some while ago are talking to a young well-dressed man. Donna looks much the same as you remember her, but Charles is sporting a bandage on his head, and one of his arms looks like it's in a sling. When they see you, Donna points excitedly in your direction, and the entire group moves to meet you.`
```
The player may not return to Earth for several years.

Reply:
Some while ago is correct, but `who` should definitely be in there.